### DISCUSSION TITLE
Do you really need .NET Framework 3.5?

### DISCUSSION THREAD

**Discussion by WhatTheBlock** [WhatTheBlock/WindowsSimplify#104] [Polls]
Do you really need .NET Framework 3.5?

Are there any applications that are limited to this framework and cannot be replaced?

Please let me know.

**Comment by biennvops** [comment]
My university's exam software requires .NET Framework 3.5...

**Reply by WhatTheBlock** [reply]
For this kind of program that does not require high-performance execution, I recommend installing the old system in a virtual machine to run it.

**Comment by ifloppy** [comment]
Just install it manually

**Reply by WhatTheBlock** [reply]
If I remove WinSxS, the manual installation will fail.

**Reply by ifloppy** [reply]
Can we use Mono instead of .NET Framework?
